examples/v1.0.0/helpers.json

```diff
@@ -68,6 +68,28 @@
 			"responses": [],
 			"synced": false
 		},
+    {
+      "id": "951fc3e8-c6b6-5c19-9f69-4e7499b3127f",
+      "headers": "Authorization: Bearer wkjehbxoqnunc2k3",
+      "url": "http://echo.getpostman.com/auth/bearer",
```

Comment:
Seems that the header has the auth repeated

Reply:
Addressed in 67b33e7749dc7a0d4d2d0d55e2094fb5130da9b6.